diff --git a/build.py b/build.py
index 497feda..64477f1 100644
--- a/build.py
+++ b/build.py
@@ -10,9 +10,12 @@ import subprocess
 import sys
 
 OPTION_SANITIZE=None
-OPTION_RELEASE_FLAGS_GCC_CLANG='-DNDEBUG -O3 -march=x86-64-v3 -mtune=corei7 -ffunction-sections -fdata-sections'
+OPTION_RELEASE_FLAGS_GCC_CLANG='-DNDEBUG -O3 -ffunction-sections -fdata-sections'
 OPTION_RELEASE_FLAGS_MSVC=None
 
+if platform.machine() in [ 'x86_64', 'amd64', 'x64' ]:
+    OPTION_RELEASE_FLAGS_GCC_CLANG += ' -march=x86-64-v3 -mtune=corei7'
+
 script_path = os.path.dirname(os.path.abspath(__file__))
 
 sysargs = sys.argv[:]
